From 29f71f2df3ba083fb59c9776af930e05f24981c9 Mon Sep 17 00:00:00 2001
From: Sivasubramanian Kannappan <skannappan@maxlinear.com>
Date: Tue, 22 Oct 2024 08:45:50 +0530
Subject: [PATCH] WLANRTSYS-85284 - Enable UL HE MUMIMO by default

Request:
   Enable by default UL HE MUMIMO

Solution:
  Change ap_config default values to enable it
---
 src/ap/ap_config.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ap/ap_config.c b/src/ap/ap_config.c
index 1303bba6b..fa7e6442e 100644
--- a/src/ap/ap_config.c
+++ b/src/ap/ap_config.c
@@ -305,7 +305,7 @@ struct hostapd_config * hostapd_config_defaults(void)
 	conf->sPowerSelection = 0; /* 100% */
 	conf->sInterferDetThresh = INTERFERENCE_DET_THRESH_DEFAULT;
 	conf->sDynamicMuTypeDownLink = 3;
-	conf->sDynamicMuTypeUpLink = 1;
+	conf->sDynamicMuTypeUpLink = 3;
 	conf->sDynamicMuMinStationsInGroup = 2;
 	conf->sDynamicMuMaxStationsInGroup = 4;
 	conf->sDynamicMuCdbConfig = 0;
-- 
2.43.0

